cli/src/client/tools.ts

```diff
@@ -82,9 +87,11 @@ export async function callTool(
   client: Client,
   name: string,
   args: Record<string, JsonValue>,
+  generalMetaData?: Record<string, string>,
```

Comment:
I think this can just be `metaData` like everywhere else. 

It seems like a good idea if different tools want something extra in the _meta field, but there's no way to exercise it from the client, so it's just more complicated code in this function.